src/components/AreaOfSpecialization/AreaOfSpecialization.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { IJobSeekerProps } from '@allTypes/reduxTypes/areaSpecializationTypes'
+import {
+  area,
+  educationLevel,
+  expectedSalary,
+  experience,
+  schedule,
+  type,
+  workplace,
+} from '@constants/filTheForm'
+import { Translation } from '@constants/translations'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { Button } from '@uiComponents/Button'
+import { Select } from '@uiComponents/Select'
+import { jobSeekerValidationSchema } from '@validation/jobSeeker/jobSeeker'
+
+interface IAreaOfSpecialization {
+  setSelectedIndex: (prev: (prev: number) => number) => void
+  setJobSeekerReview: (value: IJobSeekerProps) => void
+  jobSeekerReview: IJobSeekerProps | null
+}
+
+const defaultValues = {
+  type: type[0]?.name,
+  area: area[0]?.name,
+  educationLevel: educationLevel[0]?.name,
+  experience: experience[0]?.name,
+  schedule: schedule[0]?.name,
+  workplace: workplace[0]?.name,
+  expectedSalary: expectedSalary[0]?.name,
+}
+
+export const AreaOfSpecialization = ({
+  setSelectedIndex,
+  setJobSeekerReview,
+  jobSeekerReview,
+}: IAreaOfSpecialization) => {
+  const [t] = useTranslation()
+
+  const methods = useForm({
+    defaultValues: jobSeekerReview ?? defaultValues,
+    mode: 'onSubmit',
+    resolver: yupResolver(jobSeekerValidationSchema),
+  })
+
+  const { handleSubmit } = methods
+
+  const onSubmit = (data: IJobSeekerProps) => {
+    const areaSpecialization = {
+      ...data,
+      isActive: true,
+      postedAt: '2023-05-02',
+      status: 'Pending',
```

Comment:
Remove this, BBE will be updated accordingly

Reply:
in this case we need